doc/README.md

```diff
@@ -0,0 +1,136 @@
+### Adding GRPC Support to Tailcall Overview
+
+This proposal outlines an approach for adding GRPC client support to Tailcall to allow calling GRPC services from Tailcall resolvers.
+
+
+## Design Goals
+
+The primary design goals are:
+
+- Call GRPC services from Tailcall resolvers
+- Generate GRPC client stubs from GraphQL schema
+- Map between GraphQL and GRPC requests/responses
+- No changes needed to Tailcall server
+
+
+## Proposed Solution
+
+The proposed solution is to:
+
+Add a grpc section to the Tailcall config file, similar to http to configure GRPC endpoints:
+
+```bash
+grpc
+  endpoints = [
+    {port = 50051, proto="user.proto"},
+    {port = 50052, proto="product.proto"}
+  ]
+```
```

Comment:
Who specified this? This in my opinion needs to be autogenerated at startup time.